src/components/Profile.js

```diff
@@ -59,9 +59,18 @@ class Profile extends Component {
 
     render() {
         const { userinfo } = this.props
-        {
-            userinfo && userinfo.length === 1 ? this.state.update = true : this.state.update = false
+        
+        if (userinfo && userinfo.length === 1) {
+            this.setState({
+                update : true
+            })
+        } else {
+            this.setState({
+                update: false
+            })
         }
+            // userinfo && userinfo.length === 1 ? this.state.update = true : this.state.update = false
```

Comment:
what is this lane about 🤔

Reply:
This warning was like we have to use setState() to mutate the props so I commented out this code and use the setState() to remove the warning

Reply:
@sagar-barapatre I don't think its a good idea to use `setState` inside `render`. It [not advisable](https://stackoverflow.com/a/48226704/5182477)

This code should be modified in a way so that we don't have to mutate state inside render. For now I think you should revert the changes and disable eslint to supress the warnings. We can create another issue to fix it.

Reply:
In warning it was specifically mentioned to use setState() to remove that